src/lib/dirsize.c

```diff
@@ -96,6 +97,13 @@ double libreport_get_dirsize_find_largest_dir(
             log_debug("Skipping %s': sosreport is being generated.", dname);
             goto next;
         }
+        if (lstat(libreport_concat_path_file(dname, ".lock"), &statbuf) == 0)
+        {
+            log_warning("Skipping %s: directory locked. Is a backtrace being generated?", dname);
+            double sz = libreport_get_dirsize(dname);
+            size += sz;
```

Comment:
Why is this needed if we’re skipping the directory? I’m looking at the sosreport-still-being-generated case above and it doesn’t do that.

Reply:
I thought the directory's size still counts to the total size of the directory where problem directories are stored `/var/spool/abrt`, even if we're skipping it for now.

Reply:
Yeah, that makes sense. Do you think we should do the same above, then?

Reply:
*Ideally*, we would fold the two cases into yours, but that might be more involved, given that sosreport generation comes from an event handler.

Reply:
> Yeah, that makes sense. Do you think we should do the same above, then?  
  
Yeah, makes sense to me.